src/tests/page_objects/planner/planner.page.ts

```diff
@@ -11,7 +11,7 @@ export class PlannerPage extends AppPage {
   inlineQuickAdd =  new planner.WorkItemInlineQuickAdd($('#workItemList_quickAdd_inline'));
   sidePanel = new planner.SidePanel($('aside.f8-sidepanel'));
   quickPreview = new planner.WorkItemQuickPreview($('work-item-detail'));
-  header = new planner.ToolbarHeader($('#header-div'));
+  header = new planner.ToolbarHeader($('toolbar-panel'));
```

Comment:
@divyanshiGupta please include #header-div here as this include all the active filters too, because of this change 2tests are failing..please add it back